packages/server/graphql/mutations/helpers/sendMeetingStartToSegment.ts

```diff
@@ -8,7 +8,7 @@ const sendMeetingStartToSegment = async (newMeeting: Meeting, template?: Meeting
     userId: facilitatorUserId,
     event: 'Meeting Started',
     properties: {
-      hasIcebreaker: phases[0].phaseType === 'checkin',
+      hasIcebreaker: phases[0]!.phaseType === 'checkin',
```

Comment:
Why not stick with `?` here?

Reply:
Why not `?`?

Reply:
Thanks, @Dschoordsch, I missed that one. Update pushed! I'll move it to Maintainer Review